documentation/website/src/handbook/integration/serenity-2-migration-guide.md

```diff
@@ -0,0 +1,688 @@
+Note that Mocha is not useable yet with 2.x, so if you previously used mocha you'll need to switch to an alterntive runner for the time being. 
+Jasmine is recommended as the switch to it should be trivial. There are very few differences between how mocha and jasmine specs now look, 
+given that assertions for all runners will use the new `@serenity-js/assertions`, exposing a uniform `Ensure` API.
+So if you previously used mocha then remove your `mochaOpts` section:
+
+```javascript
+ mochaOpts: {
+    ui: "bdd",
+    compiler: "ts:ts-node/register",
+  },
+```
+
+#### Cucumber
+If you use Cucumber, you'll need a section similar to this: 
+```javascript
+ cucumberOpts: {
+        require: [
+            'features/step_definitions/**/*.ts',
+            'features/support/*.ts',
+        ],
+        'require-module': ['ts-node/register'],
+        tags: [],
+    },
+```
+Make sure you add a folder (e.g. support) and reference a setup.ts file this within the require array section where you can add any global configuration and setup (explained below)
+#### Jasmine
+If you use Jasmine, you'll need a section similar to this:
+```javascript
+ jasmineNodeOpts: {
+    requires: [
+      "ts-node/register",
+      path.resolve(__dirname, "node_modules/@serenity-js/jasmine"),
+    ],
+    helpers: [
+      "spec/support/*.ts"
+    ]
+  },
+```
+Make sure you add a folder (e.g. support) and reference a setup.ts file this within the require array section where you can add any global configuration and setup (explained in next section)
+
+### Creating a Dressing Room for the Actors
+
+If you had a `Cast` implementation that set up the default abilities on your actors like this in your 1.x project:
+```javascript
+import { protractor } from "protractor";
+import { TakeNotes } from "serenity-js/lib/screenplay";
+import { Actor, BrowseTheWeb, Cast } from "serenity-js/lib/screenplay-protractor";
+
+export class Members implements Cast {
+    public actor(name: string): Actor {
+        const notepad = {};
+        return Actor.named(name)
+            .whoCan(BrowseTheWeb.using(protractor.browser))
+            .whoCan(TakeNotes.using(notepad));
+    }
+}
+
+```
+Then this will now need to be replaced by a version 2.x equivalent which now implements `DressingRoom`:
+
+```javascript
+import { Actor, DressingRoom } from "@serenity-js/core";
+import { BrowseTheWeb } from "@serenity-js/protractor";
+import { protractor } from "protractor";
+
+export class Actors implements DressingRoom {
+  prepare(actor: Actor): Actor {
+    return actor.whoCan(
+      BrowseTheWeb.using(protractor.browser),
+      TakeNotes.usingAnEmptyNotepad(),
+    );
+  }
+}
+```
+
+The above file will need to be placed in the `support` location specified for your test runner. See previous test runner sections where the specifics of each are explained.
+    
+Create a file called `configure_stage.ts` containing the following and place in the `support` location specified for your test runner:
```

Comment:
We no longer need to interact with the `Stage` directly or assign it to `this`.

Please check out the [template repo](https://github.com/serenity-js/serenity-js-cucumber-protractor-template/blob/2946c0adb8e0018c4a50beace64b2b3377fb772a/features/step_definitions/super_calculator.steps.ts#L14)

Reply:
👍 done